example/src/screens/Casting.tsx

```diff
@@ -0,0 +1,77 @@
+import React, { useCallback } from 'react';
+import { View, StyleSheet } from 'react-native';
+import { useFocusEffect } from '@react-navigation/native';
+import {
+  Event,
+  usePlayer,
+  PlayerView,
+  SourceType,
+  BitmovinCastManager,
+} from 'bitmovin-player-react-native';
+
+function prettyPrint(header: string, obj: any) {
+  console.log(header, JSON.stringify(obj, null, 2));
+}
+
+export default function Casting() {
+  BitmovinCastManager.initialize();
+
+  const player = usePlayer();
+
+  useFocusEffect(
+    useCallback(() => {
+      player.load({
+        url: 'https://devstreaming-cdn.apple.com/videos/streaming/examples/bipbop_4x3/bipbop_4x3_variant.m3u8',
```

Comment:
I am using a cast-compatible HLS stream.

This could be changed to AOM with HLS and DASH once I have `RemoteControlConfig.prepareSource` implemented.